models/ed/R/met2model.ED2.R

```diff
@@ -27,13 +27,9 @@
 ##' @param end_date the end date of the data to be downloaded (will only use the year part of the date)
 ##' @param overwrite should existing files be overwritten
 ##' @param verbose should the function be very verbose
-met2model.ED2 <- function(in.path,in.prefix,outfolder,start_date, end_date, lst=0,overwrite=FALSE,verbose=FALSE){
+met2model.ED2 <- function(in.path,in.prefix,outfolder,start_date, end_date, lst="lst=0",..., overwrite=FALSE,verbose=FALSE){
```

Comment:
Ok, so we need to fix the problem, not the symptom. The cmdFcn should say

PEcAn.ED2::met2model.ED2('/fs/data4/NARR_CF_site_0-772/','NARR','/fs/data4/NARR_ED2_site_0-772/','1979-01-01 00:00:00','2013-12-31 00:00:00',lst = -7,overwrite = TRUE)

or

PEcAn.ED2::met2model.ED2('/fs/data4/NARR_CF_site_0-772/','NARR','/fs/data4/NARR_ED2_site_0-772/','1979-01-01 00:00:00','2013-12-31 00:00:00',lst = -7,overwrite = FALSE)

so the bit of code that builds cmdFcn needs improvement